thamos/discover.py

```diff
@@ -86,13 +200,13 @@ def discover_python_version() -> str:
     return python_version
 
 
-def discover_cpu() -> Dict[str, Union[str, int, None]]:
+def discover_cpu() -> Dict[str, str]:
```

Comment:
This function signature doesn't look right, I think it should be `Dict[str, Union[str, int]]`